src/main/java/org/springframework/data/web/ProxyingHandlerMethodArgumentResolver.java

```diff
@@ -52,9 +54,10 @@ public class ProxyingHandlerMethodArgumentResolver extends ModelAttributeMethodP
 
 	private final SpelAwareProxyProjectionFactory proxyFactory;
 	private final ObjectFactory<ConversionService> conversionService;
+	private final ProjectedPayloadDeprecationLogger deprecationLogger = new ProjectedPayloadDeprecationLogger();
```

Comment:
I have no intention of injecting any other impl in the constructor and in fact want this to be as opaque as possible as we will remove it in a few minors.